packages/frontend/src/components/NavBar/HeadwayMenuItem.tsx

```diff
@@ -64,7 +65,7 @@ const HeadwayMenuItem: FC<Props> = ({ projectUuid }) => {
     if (!isHeadwayloaded || !projectUuid) return null;
 
     return (
-        <Tooltip color="dark" label="What's new?">
+        <Tooltip color="dark" label="What's new?" withinPortal>
```

Comment:
what's this withinPortal ?

Reply:
good spot, this allows the `Button.Group` wrapper to render it properly. 
See here: https://mantine.dev/core/button/#buttongroup : "If you want to use [Tooltip](https://mantine.dev/core/tooltip/), [Popover](https://mantine.dev/core/popover/) or other overlay component, it is required to set withinPortal prop:"

So this way the `Tooltip` is rendered within a Portal and we ensure the styles applied by the `Button.Group` on the parent are added correctly. Without this `prop` the button Group wouldn't style these 4 icons correctly with nice dividers and border radius on first and last button of the group: 
![image](https://github.com/lightdash/lightdash/assets/7611706/ae7b625a-eca6-48ed-9bbc-ebbdf85e8c66)